mne/io/eeglab/eeglab.py

```diff
@@ -735,6 +740,21 @@ def _bunchify(items):
 
 
 def _read_annotations_eeglab(eeg):
+    r"""Create Annotations from EEGLAB file.
```

Comment:
Why `r`? Typically we only use this when necessary, indicating that some `\` is used in the string (almost always math or regex).

Reply:
I've no idea why this is there. But it's not the only place. 

```sh
142     r"""Read an EEGLAB .set file.
213     r"""Reader function for EEGLAB epochs files.
276     r"""Raw object from EEGLAB .set file.
421     r"""Epochs from EEGLAB .set file.
604     r"""Create events array from EEGLAB structure.
743     r"""Create Annotations from EEGLAB file.
787     r"""Create Annotations from EEGLAB file.                       
```